frontend/packages/shared/src/hooks/mutations/useCreateTextResourcesForOrgMutation.test.ts

```diff
@@ -0,0 +1,40 @@
+import { queriesMock } from 'app-shared/mocks/queriesMock';
+import { renderHookWithProviders } from 'app-development/test/mocks';
+import { useCreateTextResourcesForOrgMutation } from './useCreateTextResourcesForOrgMutation';
+import { waitFor } from '@testing-library/react';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+import { QueryKey } from 'app-shared/types/QueryKey';
+import { org } from '@studio/testing/testids';
+
+const languageMock: string = 'nb';
+
+describe('useCreateTextResourcesForOrgMutation', () => {
```

Comment:
Husk å tilbakestille mockene mellom hver test når du bruker globalt definerte mocker.
```suggestion
describe('useCreateTextResourcesForOrgMutation', () => {
  beforeEach(jest.clearAllMocks);

```